src/cascadia/WpfTerminalControl/TerminalTheme.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Terminal.Wpf
+{
+    /// <summary>

```

Comment:
For the CS code, is there any sort of StyleCop auditing we can turn on for these new projects to ensure that they start and stay in a well-formatted state? I love these comment blocks, but they don't look consistent.

Reply:
I added the stylecop.analyzers nuget package to do validation here.

Reply:
Excellent.